src/main/scala/com/monsanto/arch/cloudformation/model/resource/Lambda.scala

```diff
@@ -30,19 +29,74 @@ object Runtime {
 
     override def read(json: JsValue): Runtime = json match {
       case JsString(runtime) => new Runtime(runtime)
+      case x => deserializationError(s"Expected string for Lambda Runtime, but got $x")
     }
   }
 
 }
 
+
+/**
+  * The AWS::Lambda::Function resource creates an AWS Lambda (Lambda) function that can run code in response to events.
```

Comment:
Are these comments helpful?  I find docs get quickly out of date, and always consult the original docs anyway for any arguments.

Reply:
Yeah, it's not a perfect situation. I've found it occasionally helpful for the `TheThing: Token[String]` type parameters to see if it wants an ARN or a name, for example, without needing to swap to my browser and look it up.